devtools/gradle/src/main/resources/quarkus-platform.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.platform.json.groupId=io.quarkus
```

Comment:
what is this file representing ?

note - we use kebab-case (i.e. group-id) every where else in properties, json and yaml. Suggest we do the same here.

Reply:
It's actually not used anymore. I was thinking about externalizing the default platform coordinates for the tools this way instead of hardcoding values in Java classes.
groupId and artifactId is what's common in the Maven world. I don't think you'll find group-id and artifact-id in maven-related configs (be it XML or something else).

Reply:
not correct, thats what we have in our and even spring similar properties/yaml/json.